packages/suite-desktop/src/libs/processes/CoinjoinProcess.ts

```diff
@@ -7,6 +7,7 @@ export class CoinjoinProcess extends BaseProcess {
     constructor() {
         super('coinjoin', 'WalletWasabi.WabiSabiClientLibrary', {
             autoRestart: 0,
+            stdio: true,
```

Comment:
I'd personally pass here 
`stdio: ['inherit', 'inherit', ''inherit']`
and assign `['ignore', 'ignore', 'ignore']` as a default in BaseProcess.

Reply:
done